examples/inference/lisa_smbhb_inj/lisa_smbhb_relbin.ini

```diff
@@ -50,6 +50,7 @@ spin2z = 0.36905807298613247
 distance = 17758.367941273442
 inclination = 1.5970175301911231
 t_obs_start = 31536000
+delta_f = 3.1709791983764586e-08
```

Comment:
You should never be setting this as it is not your choice but the analysis which must determine the delta_f it wants to use.

Reply:
I see. So this should really be set and checked like the f_lower check in relbin.py instead of a static parameter here?

Reply:
@ConWea I don't think I follow. Maybe it would help to explain the motivation behind this suggestion.

Reply:
@ahnitz It seems Connor also pushed changes from this PR https://github.com/gwastro/pycbc/pull/4584 to here.

Reply:
@ahnitz I was thinking that because BBHX plugin needs delta_f to be set (or you get the error that @WuShichao showed in the other PR), that it should be set in waveform.py like the f_lower check, and set it to -1. Otherwise it isn't set anywhere and throws an error when trying to produce the waveform without delta_f.

Reply:
@ConWea See comments on that section.